libs/labelbox/src/labelbox/schema/internal/data_row_uploader.py

```diff
@@ -0,0 +1,294 @@
+import json
+import os
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from typing import Iterable, List, Union
+
+from labelbox.exceptions import InvalidQueryError
+from labelbox.exceptions import InvalidAttributeError
+from labelbox.exceptions import MalformedQueryException
+from labelbox.orm.model import Entity
+from labelbox.orm.model import Field
+from labelbox.schema.embedding import EmbeddingVector
+from labelbox.schema.internal.data_row_create_upsert import DataRowItemBase
+from labelbox.schema.internal.datarow_upload_constants import MAX_DATAROW_PER_API_OPERATION
+
+
+class UploadManifest:
+
+    def __init__(self, source: str, item_count: int, chunk_uris: List[str]):
+        self.source = source
+        self.item_count = item_count
+        self.chunk_uris = chunk_uris
+
+    def to_dict(self):
+        return {
+            "source": self.source,
+            "item_count": self.item_count,
+            "chunk_uris": self.chunk_uris
+        }
+
+
+class DataRowUploader:
```

Comment:
Correct me if i'm wrong, but this class has only static methods, no instance variables, and is never instantiated? If so, I think we don't need the class. Just export its static methods as functions directly

Reply:
I am just trying to follow what seems to be an sdk pattern: not to use functions, but to group like functions in a class... lmk if my understanding is not correct